runtime/compiler/control/CompilationThread.cpp

```diff
@@ -9366,6 +9779,8 @@ void *
 TR::CompilationInfo::compilationEnd(J9VMThread * vmThread, TR::IlGeneratorMethodDetails & details, J9JITConfig *jitConfig, void *startPC,
                                    void *oldStartPC, TR_FrontEnd *fe, TR_MethodToBeCompiled *entry, TR::Compilation *comp)
    {
+   if (!comp)
+      return NULL;
```

Comment:
This seems a little odd to get all the way to `compilationEnd()` without ` comp` object. Did JITServer add something to make this possible? If its necessary maybe a comment is warranted.

Reply:
This was a change done by Noah in commit https://github.com/eclipse/openj9/commit/eaaac12c9bff926ebda1e2286327a74a49354ddf
"Prevent crash when compiling non-ordinary methods during shutdown"
I'll add a FATAL_ASSERT because I don't see how comp can be NULL.